### ISSUE TITLE
Ability to change font of input text

### ISSUE THREAD

**Issue by LukeG294** [LukeG294/TextCheck#1]
**Feature Request/Idea:**

A dropdown menu to change the font of the text in the text box. The goal of this is for the font to be exported to be used in other sites.
Currently, only one font is available. Feel free to experiment with the current version of the extension by downloading it at https://chrome.google.com/webstore/detail/textcheck/lakkaggkknfcppjimmjacjconjfnadif

![image](https://user-images.githubusercontent.com/62516707/96377199-d4257f00-1138-11eb-9fce-933b1820493b.png)

**Please comment here before starting to work on this.**

Thanks!

**Comment by Aman2632** [comment]
@LukeG294 can I work on it

**Comment by LukeG294** [comment]
Yep I'll assign you now, thanks!  :)  The html code is located at https://github.com/LukeG294/TextCheck/blob/main/hello.html

**Comment by LukeG294** [comment]
Just to clarify, the goal is for the user to be able to choose between a few font types to use. I just noticed how my original comment could have been confusing.
